content/guides/additional/sets.md

```diff
@@ -0,0 +1,230 @@
++++
+title = "Set Identities"
+description = "Explore set relations using the Hoon standard library"
+weight = 190
++++
+
+#   Set Identities and Relations
+
+While the developer documentation on `$set`s and the `+in` core is comprehensive, it is organized alphabetically which can make it difficult to see what's going on with set relations.  This article will describe [set identities and relations](https://en.wikipedia.org/wiki/List_of_set_identities_and_relations) through the Hoon standard library.
+
+A `$set` is a tree with a particular internal order based on the hash of the value.  This tends to balance the values and make lookup and access more efficient over large sets.
+
+##  Set Creation & Membership
+
+### Define a Set
+
+![](https://media.urbit.org/docs/hoon-syntax/set-identity.png)
+
+[`++silt`](https://developers.urbit.org/reference/hoon/stdlib/2l#silt) produces a `$set` from a `$list`.
+
+```hoon
+> `(set @tas)`(silt `(list @tas)`~[%a %b %c %a])
+{%b %a %c}
+```
+
+### Add Members
+
+![](https://media.urbit.org/docs/hoon-syntax/set-addition.png)
+
+[`++put:in`](https://developers.urbit.org/reference/hoon/stdlib/2h#putin) adds an element $x$ to a set $A$.
```

Comment:
all cases of the $something$ syntax render as-is. Is that meant to do something different?